lib/commands/pcap.js

```diff
@@ -1,11 +1,10 @@
-import Pyidevice from '../py-ios-device-client';
-import {fs, tempDir, logger, util} from 'appium/support';
+import { Pyidevice } from '../real-device-clients/py-ios-device-client';
```

Comment:
spaces?

Reply:
Hm, we mixed the space stuff for import. Can be ignored.

Reply:
I assume this is because of VSCode. Imports that it adds automatically do have spaces. Eventually I don't find that so important. In the worst case we could always run prettify in a separate PR